esp/esp/program/modules/forms/teacherreg.py

```diff
@@ -91,7 +90,6 @@ class TeacherClassRegForm(FormWithRequiredCss):
     class_size_max = forms.ChoiceField( label='Maximum Number of Students',
                                         choices=[(0, 0)],
                                         widget=BlankSelectWidget(),
-                                        validators=[validators.MinValueValidator(1)],
```

Comment:
Why are we removing this validator?

Reply:
`ChoiceField`s coerce the selection to a string, which causes an error during validation because you can't compare strings and integers. In this case, since it uses a select widget, you shouldn't be able to submit a non-positive value anyways, so I think it is ok to remove?